frontend/src/scenes/sceneLogic.ts

```diff
@@ -267,6 +267,8 @@ export const sceneLogic = kea<sceneLogicType>({
             const wasNotLoaded = !loadedScene
 
             if (!loadedScene) {
+                // if we can't load the scene in a second, show a spinner
+                const timeout = window.setTimeout(() => actions.setScene(scene, params, true), 1000)
```

Comment:
It's tempting to bike-shed the length of time http://www.exactlywhatistime.com/psychology-of-time/time-perception/ and push it down to 500

But LGTM :)

Reply:
Sure, made it even snazzier ✨